metadata-ingestion/src/datahub/sdk/_shared.py

```diff
@@ -578,3 +606,117 @@ def remove_link(self, link: LinkInputType) -> None:
             self._link_key,
             self._parse_link_association_class(link),
         )
+
+
+class HasVersion(Entity):
+    """Mixin for entities that have version properties."""
+
+    def _get_urn_name(self) -> str:
+        if hasattr(self.urn, "name"):
+            return self.urn.name
+        return "unknown"  # Fallback if name isn't available
```

Comment:
this feels a bit janky

we can have an abstract method that the subclasses are required to implement. in the patch interface, `HasCustomPropertiesPatch` does something similar

Reply:
replaced this to just use GUID like we do in mlflow UI (feedback by Andrew) https://github.com/datahub-project/datahub/pull/12587/files#diff-bf0e8f53a0658f4ffd5004b9f0be1b7a4220b19c6750c39b955b7a43dc276d27R501